internal/install/bundle_installer.go

```diff
@@ -105,9 +105,11 @@ func (bi *BundleInstaller) InstallContinueOnError(bundle *recipes.Bundle, assume
 
 func (bi *BundleInstaller) reportBundleStatus(bundle *recipes.Bundle) {
 	for _, recipe := range bundle.BundleRecipes {
-		for _, ds := range recipe.DetectedStatuses {
-			e := execution.RecipeStatusEvent{Recipe: *recipe.Recipe, ValidationDurationMs: ds.DurationMs}
-			bi.statusReporter.ReportStatus(ds.Status, e)
+		if !bi.IsRecipeInstalled(recipe.Recipe.Name) {
```

Comment:
@NRhzhao this change seems independent of this work. Is this a needed bug fix? What's the use case illustrating that we don't want to fire detected if the recipe has been installed?

Reply:
@Julien4218  Hi Julien, since we doing detection for all recipes during additional bundling.  If we re-report installed recipes status (infra, log), it reset the NR1 UI.
The alternative is to filter out the installed recipes during bundling by passing in installed recipes.  
Let me know if you would prefer doing that way instead.

Reply:
I see @NRhzhao 
Given we want to detect all the time now (guided+targeted) with the exception of the `-c` mode, would it be easier if we were to move all the detection out of the bundling (before any), perhaps right after we do all the validation (NR connect OK)?